test/chi/components/card.pug

```diff
@@ -159,7 +159,7 @@ title: Card
   h3 Empty
   .chi-card.-empty(style="width:20rem;")
     .chi-card__content
-      i.chi-icon.-md.-text--muted(class ='icon-circle-plus-outline')
+      i.chi-icon.-md.-icon--muted(class ='icon-circle-plus-outline')
```

Comment:
@mattnickles we can remove the icon and change the caption to: This is an empty card, to be consistent with doc page